packages/kit/src/__typetests__/decompile-transaction-message-fetching-lookup-tables-typetest.ts

```diff
@@ -0,0 +1,80 @@
+import {
+    CompiledTransactionMessage,
+    CompiledTransactionMessageWithLifetime,
+    decompileTransactionMessageFetchingLookupTables,
+    GetMultipleAccountsApi,
+    isTransactionMessageWithBlockhashLifetime,
+    Rpc,
+    TransactionMessage,
+    TransactionMessageWithBlockhashLifetime,
+    TransactionMessageWithFeePayer,
+    TransactionMessageWithLifetime,
+} from '..';
```

Comment:
As in the first PR in the stack, change this to import from the relevant modules.